flaml/automl.py

```diff
@@ -831,7 +827,7 @@ def _validate_ts_data(
             logger.warning(f"Removed duplicate rows based on all columns")
             assert (
```

Comment:
Please change assert to warning, and add a test to demonstrate that forecasting with regression can handle multiple time-series in the datasets, i.e., with duplicated timestamp but with different values on some other attribute.

Reply:
We decided that flaml will not support forecasting with regression when there are duplicated timestamps but different values on some other attribute. The only known estimator that was capable of it was prophet and based on a benchmark test, the performance was suboptimal. This is why an error is prompted.

Reply:
@sonichi

Reply:
@int-chaos we won't handle duplicate timestamps when there is only one time series. But we do want to handle multiple time series indicated by categorical columns.